src/analyses/invariant_set.h

```diff
@@ -22,6 +22,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "interval_template.h"
 
+#define nullptr_exceptiont(str) str
```

Comment:
This is temporary, until #1063 gets merged. Of course this should be a custom exception type derived from `invariant_failedt`, and these common exception types should probably live in `util`. Should I add a "util/standard_exceptions.h" header and just put macros in it for now?